ubuntudesign/vanilla-brochure-theme
# Build the media object pattern

## Done
Built the media object pattern to match the design provided.  Added the example and documentation for the media-object.

Reorganised the _theme and build scss files to match our standard.

## QA
- Run `gulp develop`
- Open `examples/_patterns_media-object.html` in a browser
- Check it matches the design in the issue
- Check the documentation is ok

## Details
Fixes https://github.com/ubuntudesign/vanilla-brochure-theme/issues/7

## Screenshots
![vanilla brochure theme media object](https://cloud.githubusercontent.com/assets/1413534/22559969/c20cea28-e96a-11e6-8780-a2611ce2987c.png)
